tests/conftest.py

```diff
@@ -150,5 +150,36 @@ def empty_metadata_object():
         is_withdrawn=0
     )
 
+@pytest.fixture(scope="session")
+def metadata_with_tex():
+    return Metadata(
+        metadata_id=1,
+        document_id=1,
+        paper_id="1234.56789",
+        created=datetime(2023,1,1,10,3,6),
+        updated=datetime(2023,1,1,15,7,8),
+        submitter_id=42,
+        submitter_name="John Doe",
+        submitter_email="john.doe@example.com",
+        source_size=1024,
+        source_format="pdf",
+        source_flags=None,
+        title="Un th\'eor\`eme sur les chats",
```

Comment:
Title will incorrectly escape in python.
Change
title="Un th\'eor\`eme sur les chats",
to 
title="Un th\\'eor\`eme sur les chats",